### PR TITLE
Update ASR and General Docs

### PR DESCRIPTION
**Pull request by blisc**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -64,7 +64,7 @@ THE LATEST STABLE VERSION OF NeMo is **0.9.0** (which is available via PIP).
 **Requirements**
 
 1) Python 3.6 or 3.7
-2) PyTorch 1.2.* or 1.3.* with GPU support
+2) PyTorch 1.4* with GPU support
```
Comment (tkornuta-nvidia) [inline README.rst]: I know I know - we are NVIDIA :)

But still, NeMo can work without GPUs, so can we say here: "(with or without GPU support)

File: docs/sources/source/asr/tutorial.rst [hunk 2]
```diff
@@ -42,7 +45,10 @@ We will be using an open-source LibriSpeech :cite:`asr-tut-panayotov2015librispe
     # python get_librispeech_data.py --data_root=data --data_set=ALL
 
 .. note::
-    You should have at least 26GB of disk space available if you've used ``--data_set=dev_clean,train_clean_100``; and at least 110GB if you used ``--data_set=ALL``. Also, it will take some time to download and process, so go grab a coffee.
+    You should have at least 52GB of disk space available if you've used ``--data_set=dev_clean,train_clean_100``; and
+    at least 250GB if you used ``--data_set=ALL``. Also, it will take some time to download and process, so go grab a
+    coffee. After downloading, you can remove the original .tar.gz archives and .flac files to cut the disk usage in
```
Comment (tkornuta-nvidia) [inline docs/sources/source/asr/tutorial.rst]: eheheh, nice!

File: docs/sources/source/asr/tutorial.rst [hunk 3]
```diff
@@ -260,19 +267,19 @@ Enabling multi-GPU training with NeMo is easy:
...
 
-    python -m torch.distributed.launch --nproc_per_node=<num_gpus> <nemo_git_repo_root>/examples/asr/jasper.py ...
+    python -m torch.distributed.launch --nproc_per_node=<num_gpus> <nemo_git_repo_root>/examples/asr/quartznet.py ...
 
 
 Large Training Example
 ~~~~~~~~~~~~~~~~~~~~~~
 
-Please refer to the `<nemo_git_repo_root>/examples/asr/jasper.py` for comprehensive example. It builds one train DAG and up to three validation DAGs to evaluate on different datasets.
+Please refer to the `<nemo_git_repo_root>/examples/asr/quartznet.py` for comprehensive example. It builds one train DAG and up to three validation DAGs to evaluate on different datasets.
```
Comment (tkornuta-nvidia) [inline docs/sources/source/asr/tutorial.rst]: "and up to three validation DAGs to evaluate on different datasets." -> "and **uses** up to three validation DAGs to evaluate on different datasets."

But aside of that - what does it mean? How many Validation DAGs do we have in that script? Cannot we have four?

File: docs/sources/source/index.rst [hunk 4]
```diff
@@ -78,49 +82,32 @@ then you can simply use pip to install the latest released version (**currently
...
-    pip install nemo-asr # installs NeMo ASR collection
-    pip install nemo-nlp # installs NeMo NLP collection
-    pip install nemo-tts # installs NeMo TTS collection
+    pip install nemo_toolkit[all] # Installs NeMo Core and all collections including nemo_asr, nemo_nlp, nemo_tts
 
-**Tutorials**
+Tutorials
+---------
 
 * `Speech recognition <https://nvidia.github.io/NeMo/asr/intro.html>`_
```
Comment (tkornuta-nvidia) [inline docs/sources/source/index.rst]: Shouldn't we use RST/internal links instead?

File: README.rst (hunk 1 above)
Comment (blisc) [reply README.rst]: I think that is a decision you should ask Poonam or Negar to comment on

File: docs/sources/source/asr/tutorial.rst (hunk 3 above)
Comment (blisc) [reply docs/sources/source/asr/tutorial.rst]: Updated

File: docs/sources/source/index.rst (hunk 4 above)
Comment (blisc) [reply docs/sources/source/index.rst]: You're probably right. Let me see if I can amend this

Comment (blisc) [reply docs/sources/source/index.rst]: updated

File: docs/sources/source/asr/tutorial.rst (hunk 3 above)
Comment (tkornuta-nvidia) [reply docs/sources/source/asr/tutorial.rst]: just noticed: "builds one trains"

**Pull request merged**